src/audio/audio.cpp

```diff
@@ -36,6 +37,37 @@
 #endif
 
 /**
+@internal
+
+@class Audio::Private
+
+@brief Encapsulates private audio framework from public qTox Audio API.
+*/
+class Audio::Private
+{
+public:
+    Private()
+        : minInputGain{-30.0}
+        , maxInputGain{30.0}
+        , gain{0.0}
+        , gainDecibel{0.0}
+    {
+    }
+
+    void setInputGain(qreal dB)
+    {
+        gainDecibel = qBound(minInputGain, dB, maxInputGain);
+        gain = qPow(10.0, (dB * 0.05));
+    }
+
+public:
```

Comment:
>Keep these fields public somewhat defeats the above function. …

Yep, we can make them immutable by adding one or two more wrapper functions to completely control this throughout qTox Audio API.